util/linux_install.sh

```diff
@@ -96,9 +96,13 @@ elif grep ID /etc/os-release | grep -q gentoo; then
 	fi
 
 elif grep ID /etc/os-release | grep -qE "opensuse|tumbleweed"; then
+	CROSS_AVR_GCC=cross-avr-gcc8
```

Comment:
would `cross-avr-gcc` work better here?

Reply:
Sorry, no, I mean for the actual package name.

Reply:
I don't know about tumbleweed, but openSUSE 15 has no cross-avr-gcc package available in the default repositories.

Reply:
cross-avr-gcc8 was picked by whoever added the openSUSE support

Reply:
Okay, thanks for the clarification.  That's basically what I was asking.